### PR TITLE
Eg smart string fix

### PR DESCRIPTION
**Pull request by egolol**

### CODE REVIEW

**Comment by annacanal** [comment]
When I calculate Degree, and then I press show, I get an error. 
Also when pressing the analysis button to close all subfigures, degree stays there and an error jumps. Not sure if this was supposed to be fixed here

**Comment by annacanal** [comment]
Screenshot 2022-02-25 at 14 56 10
enlarge G is nicer. But the adjacency matrix plot and the graph button are overlapping with the table, I remember from yesterday they were not.

**Comment by annacanal** [comment]
> When I calculate Degree, and then I press show, I get an error. Also when pressing the analysis button to close all subfigures, degree stays there and an error jumps. Not sure if this was supposed to be fixed here

`Error using matlab.ui.Figure/get
Invalid or deleted object.

Error in PPAnalyzeGroup_G/cb_measure_gui (line 1077)
		            tmp_gui = get(tmp_f, 'UserData');

Error in PPAnalyzeGroup_G/draw/cb_measure_plot_btn (line 833)
		            pr.cb_measure_gui();
 
Error while evaluating UIControl Callback.`

**Comment by annacanal** [comment]
This error jumps when I successfully calculated a measure, showed it, and when I press Analysis to closes everything:

```
Error using Degree.existsTag (line 413)
BRAPH2:Degree:WrongInput The value PE is not a valid tag for Degree

Error in Degree.getPropProp (line 444)
				Degree.existsTag(tag);

Error in Element/get (line 845)
            prop = el.getPropProp(pointer);

Error in PPAnalyzeGroup_G/cb_hide (line 1375)
		            pe = gui.get('PE');

Error in GUI/cb_hide (line 1598)
		        pr.cb_hide()

Error in PPPipeline_PSDict/update (line 847)
		                            gui.cb_hide()

Error in PPPipeline_PSDict/draw/cb_pc_btn (line 766)
		        pr.update('Section', s, 'Code', c)
 
Error while evaluating UIControl Callback.
```

**Comment by annacanal** [comment]
When I press 'C' at the Graph measure comparisons, this error jumps:
```

Unrecognized method, property, or field 'adj_plot_tgl_btn' for class 'PPCompareGroup_CPDict_WU'.

Error in PPCompareGroup_CPDict_WU/update (line 806)
		        set(pr.adj_plot_tgl_btn, 'Visible', 'on')

Error in PlotElement/update (line 1126)
		        pe.get('PR_DICT').getItem(prop).update()

Error in PlotProp/cb_button_calc (line 1152)
		    pe.update()

Error in PlotProp/draw/cb_button_calc (line 990)
		        pr.cb_button_calc()
 
Error while evaluating UIControl Callback.
```

File: braph2genesis/src/analysis/_PPAnalyzeGroup_G.gen.m [hunk 1]
```diff
@@ -65,7 +65,7 @@
             'Units', 'normalized', ...
             'String', 'Graph', ...
             'TooltipString', 'Open the graph class GUI.', ...
-            'Position', [.79 .71 .2 .2], ...
+            'Position', [.79 .71 .2 .15], ...
```
Comment (annacanal) [inline braph2genesis/src/analysis/_PPAnalyzeGroup_G.gen.m:68]: 'Position', [.79 **.77** .2 .15], ...
makes that the buttons have space with the table under

File: braph2genesis/src/analysis/_PPAnalyzeGroup_G.gen.m [hunk 2]
```diff
@@ -76,7 +76,7 @@
             'Units', 'normalized', ...
             'CData', imresize(imread('icon_plot_adj.png'), [40 40]), ...
             'TooltipString', 'Plot to adjacency matrix plot.', ...
-            'Position', [.01 .71 .2 .2], ...
+            'Position', [.01 .71 .2 .15], ...
```
Comment (annacanal) [inline braph2genesis/src/analysis/_PPAnalyzeGroup_G.gen.m:79]: 'Position', [.01 **.77** .2 .15], ...
makes that the buttons have space with the table under

File: braph2genesis/src/analysis/_PPAnalyzeGroup_G.gen.m [hunk 3]
```diff
@@ -87,7 +87,7 @@
             'Units', 'normalized', ...
             'CData', imresize(imread('icon_plot_lines.png'), [40 40]), ...
             'TooltipString', 'Plot to line plot.', ...
-            'Position', [.23 .71 .2 .2], ...
+            'Position', [.23 .71 .2 .15], ...
```
Comment (annacanal) [inline braph2genesis/src/analysis/_PPAnalyzeGroup_G.gen.m:90]: 'Position', [.23 **.77** .2 .15], ...
makes that the buttons have space with the table under

**Review by annacanal — CHANGES_REQUESTED** [review]

**Comment by egolol** [comment]
solved

**Comment by c-yuwei** [comment]
> Screenshot 2022-02-25 at 14 56 10 enlarge G is nicer. But the adjacency matrix plot and the graph button are overlapping with the table, I remember from yesterday they were not.

solved

**Comment by c-yuwei** [comment]
For @annacanal , please have Daniel's computer for testing the compiling issue
image

**Review by giovannivolpe — APPROVED** [review]

**Pull request merged**